lib/periodicChecks.js

```diff
@@ -26,6 +26,7 @@ const {
   getAllProjectCards,
   hasPendingReviews
 } = require('./utils');
+const utilityModule = require('./utils');
```

Comment:
Can you include the `MIN_BUILD_DATE` in the import aboce this?

Reply:
The issue with that is I won't be able to mock it in the tests.

Reply:
Ah, I see, okay.